mocha.el

```diff
@@ -54,21 +53,23 @@ If MOCHA-FILE is specified run just that file otherwise run MOCHA-PROJECT-TEST-D
           mocha-options " "
           (or mocha-file mocha-project-test-directory)))
 
-;;;###autoload
 (defun mocha-run (&optional mocha-file)
   "Run mocha in a compilation buffer.
 
 If MOCHA-FILE is specified run just that file otherwise run MOCHA-PROJECT-TEST-DIRECTORY"
   (compile (mocha-command mocha-file))
-  (kill-buffer "*mocha tests*")
+  (when (get-buffer "*mocha tests*")
+    (kill-buffer "*mocha tests*"))
```

Comment:
Cool, my impression from reading the manual was that `kill-buffer` was a noop if the buffer didn't exist.